### ISSUE TITLE
Can SMART-Driver be used if the OS lives on the external drive?

### ISSUE THREAD

**Issue by daniels** [kasbert/OS-X-SAT-SMART-Driver#47]
I use an USB SSD as boot disk. Would it be possible to install the SAT-SMART-Driver in the OS that lives on that disk to get SMART status from the same disk? (I guess no ...)

**Comment by kasbert** [comment]
No. This driver is for external enclosures only. If your internal SSD doesn't show smart status,  this driver does not helposti you.

**Comment by kasbert** [comment]
Sorry. Ihan didn't read tour description well. Also The mobile autocorrect IS mocking me. I have to think about it.
